src/Components/Aspire.Azure.Messaging.EventHubs/EventHubsComponent.cs

```diff
@@ -101,4 +105,41 @@ protected static string GetNamespaceFromSettings(AzureMessagingEventHubsSettings
                 $"the '{configurationSectionName}' configuration section.");
         }
     }
+
+    protected sealed class AzureEventHubHealthCheck : IHealthCheck, IAsyncDisposable
```

Comment:
Is there a reason we aren't using https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/tree/master/src/HealthChecks.Azure.Messaging.EventHubs? Typically we try to use those because they are the "best practice", and if they aren't meeting our needs, we should update them to meet our needs.

That way any improvements can be used by anyone - not just Aspire.

Reply:
I knew about these packages but when I looked for an eventhubs one, it was not in the public feed. I am confused by this new information.

Reply:
Each package needs to be added, I will do that for this one.

Reply:
If MSFT have official HC packages you should be using them in all components. This doesn't seem to be the case...

Reply:
Done

Reply:
> If MSFT have official HC packages you should be using them in all components. This doesn't seem to be the case...

Microsoft doesn't have any "official health check" packages. .NET Aspire has been using https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks. See https://github.com/dotnet/aspire/tree/main/src/Components#health-checks

Reply:
I should be able to switch to using the 3rd party package some time this week.

Reply:
Were you able to use the package from https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks?

Reply:
> Were you able to use the package from https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks?

Doing it right at this moment.

Reply:
See https://github.com/dotnet/aspire/issues/4332

Reply:
Okay, so nobody has an answer for that. Can I consider my work here _done_ @eerhardt ?

Reply:
You shouldn't be doing anything special with unshipped APIs right now. What you currently have changed in the file looks good.

Reply:
Alright, then merge at your leisure.